omod_1.x/src/main/java/org/openmrs/module/openhmis/cashier/web/controller/JasperReportController.java

```diff
@@ -51,8 +52,8 @@ public String render(@RequestParam(value = "reportId", required = true) int repo
 		JasperReportService jasperService = Context.getService(JasperReportService.class);
 		JasperReport report = jasperService.getJasperReport(reportId);
 		if (report == null) {
-			response.sendError(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, "Could not find report '" + reportId + "'");
-			return null;
+			message = "Could not find report '" + reportId + "'";
```

Comment:
Change this error message to: "No Jasper Report with an ID of '{ID}' could be found."